Quasardb/Query/QdbContinuousQuery.cs

```diff
@@ -61,10 +61,13 @@ internal QdbContinuousQuery(qdb_handle handle, string query, Mode mode, int refr
             var error = qdb_api.qdb_query_continuous(_handle, query, (qdb_query_continuous_mode)mode, refresh_rate_ms, _internal_callback, IntPtr.Zero, out _cont_handle);
             if (error != qdb_error.qdb_e_ok)
             {
-                qdb_sized_string error_message;
                 qdb_error err;
-                qdb_api.qdb_get_last_error(_handle, out err, out error_message);
-                throw new QdbQueryException((string)error_message ?? qdb_api.qdb_error(error));
+                qdb_sized_string* error_message = (qdb_sized_string*)IntPtr.Zero;
+                var ec = qdb_api.qdb_get_last_error(_handle, out err, out error_message);
+                QdbExceptionThrower.ThrowIfNeeded(ec);
```

Comment:
Isn't this a bit redundant, again? Because basically we're *always* throwing here, right? Or am I misunderstanding?